test/routes/article-list.test.ts

```diff
@@ -3,11 +3,13 @@ import { Next, Response } from 'koa';
 import articleList from '../../src/routes/article-list';
 import createContext from '../context';
 import runMiddleware from '../middleware';
+import InMemoryArticles from '../../src/adaptors/in-memory-articles';
 
 const makeRequest = async (next?: Next): Promise<Response> => {
   const context = createContext();
+  const articles = new InMemoryArticles();
 
-  return runMiddleware(articleList(context.router), context, next);
+  return runMiddleware(articleList(articles, context.router), context, next);
 };
 
 describe('article list', (): void => {
```

Comment:
We should expand the tests to cover the empty list, but also a list with articles.